components/internal/Table.tsx

```diff
@@ -37,7 +37,7 @@ export default function Table(props: Props) {
     if (onChangeSortOrder !== undefined) {
       onChangeSortOrder(getSortOptions(sortBy));
     }
-  }, [onChangeSortOrder, sortBy]);
+  }, [sortBy]);
```

Comment:
Any reason why `onChangeSortOrder` was removed here? I think our linting should catch that :/

Reply:
It was making the page super super super slow 🐢 this is the error i saw in the console
![image](https://user-images.githubusercontent.com/55164714/130310232-5e0b35d6-9b5b-4a93-ae15-ee0b467858d0.png)

I was act wondering why we had the sort order handler function in the dependencies to begin with?

Reply:
Interesting, that's a good catch. It didn't make any sense being in there.